### PR TITLE
BLO-27 RPC done

### PR DESCRIPTION
**Pull request by Svenmarim**

### CODE REVIEW

File: src/services/rpcTransactionService.ts [hunk 1]
```diff
@@ -29,6 +30,12 @@ export class RpcTransactionService {
     }
 
     public addTransaction(transaction: Transaction) {
-        this.client.addTransaction(transaction);
+        this.client.addTransaction(transaction, function(err: any) {
+            if (err) {
+                logger.info(err);
```
Comment (roelvdboom) [inline src/services/rpcTransactionService.ts]: Should use `logger.error()`

File: src/services/rpcTransactionService.ts [hunk 2]
```diff
@@ -29,6 +30,12 @@ export class RpcTransactionService {
...
+            } else {
+                logger.info("succed");
```
Comment (roelvdboom) [inline src/services/rpcTransactionService.ts]: Replace with `Succeed`

**Review by roelvdboom — CHANGES_REQUESTED** [review]

**Review by quintaartsen — APPROVED** [review]

**Review by quintaartsen — CHANGES_REQUESTED** [review]
fix the build errors

**Review by quintaartsen — APPROVED** [review]

File: src/services/rpcTransactionService.ts [hunk 3]
```diff
@@ -29,6 +30,12 @@ export class RpcTransactionService {
     }
 
     public addTransaction(transaction: Transaction) {
-        this.client.addTransaction(transaction);
+        this.client.addTransaction(transaction, (err: Error) => {
+            if (err) {
+                logger.error(err);
+            } else {
+                logger.info("succeed");
```
Comment (hugovandevliert) [inline src/services/rpcTransactionService.ts]: Please write a proper succeed message.

**Review by hugovandevliert — CHANGES_REQUESTED** [review]

**Review by hugovandevliert — APPROVED** [review]

**Review by roelvdboom — APPROVED** [review]

File: src/services/rpcTransactionService.ts [hunk 4]
```diff
@@ -29,6 +30,12 @@ export class RpcTransactionService {
     }
 
     public addTransaction(transaction: Transaction) {
-        this.client.addTransaction(transaction);
+        this.client.addTransaction(transaction, (err: Error) => {
+            if (err) {
```
Comment (hugovandevliert) [inline src/services/rpcTransactionService.ts:34]: Please change this if..else statement to the following:
`if (err) {
    logger.error(err);
    return;
}
logger.info("Transaction sending succeeded");`

**Review by hugovandevliert — CHANGES_REQUESTED** [review]

**Review by hugovandevliert — APPROVED** [review]

**Pull request merged**